web-portal/components/DateTimePicker.vue

```diff
@@ -77,6 +77,11 @@
                 <option value="am">AM</option>
                 <option value="pm">PM</option>
               </select>
+              <select ref="eventTimezone" name="event_timezone" v-model="event_timezone">
+                <option v-for="(tz) in this.$config.TIMEZONE_OPTIONS.split(',')" :key="tz">
```

Comment:
Small nitpick but the `this` isn't technically necessary inside the `template` -- you should be able to refer to it as just `$config.TIMEZONE_OPTIONS...`.